doc/talks/paris/slides.tex

```diff
@@ -1,7 +1,7 @@
 \documentclass[14pt]{beamer}
 \usepackage[utf8]{inputenc}
 
-\input{../macros.tex}
+\input{../../../archive/doc/macros.tex}
```

Comment:
This seems very fragile to me, it would make more sense to just copy the `macros.tex` file next to the file. In the long term non-fragile LaTeX is more important than some imagined "consistency".